README.md

```diff
@@ -0,0 +1,86 @@
+# dbt-rpc-client
+[![PyPI version](https://badge.fury.io/py/dbt-rpc-client.svg)](https://badge.fury.io/py/dbt-rpc-client)
+![PyPI - Status](https://img.shields.io/pypi/status/dbt-rpc-client)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/dbt-rpc-client)
+![PyPI - License](https://img.shields.io/pypi/l/dbt-rpc-client)
+[![Build Status](https://travis-ci.com/goodeggs/dbt-rpc-client.svg?branch=master)](https://travis-ci.com/goodeggs/dbt-rpc-client.svg?branch=master)
+
+A python SDK for interacting with a [dbt rpc server](https://docs.getdbt.com/docs/rpc).
+
+## Installation
+
+```bash
+$ pip install dbt-rpc-client
+```
+
+## Basic Usage
+
+```python
+from dbt_rpc_client import DbtRpcClient
+
+rpc = DbtRpcClient(hostname="0.0.0.0", port=8580)
+
+# Getting the current status of the dbt rpc server.
+response = rpc.status()
+assert response.ok
+
+# Polling a dbt rpc operation.
+response = rpc.poll(request_token="1234abcd")
+assert response.ok
+
+# Running dbt models via CLI command.
+response = rpc.cli(cli_args="dbt run --models @model1 +model2+")
+assert response.ok
+
+# Running dbt models via `run` method.
+response = rpc.run(models=["@model1", "+model2+"])
```

Comment:
here again -- does `rpc.run` block until the run finishes?  probably a near-real-world end-to-end example would be valuable.

Reply:
@bobzoller No, `rpc.run` doesn't block. IMO, blocking is an undesirable pattern since sometimes you may not **need** to be blocked by the completion of an operation. If you are relaying on the completion of an operation, you can always combine a `rpc.run()` with an `rpc.poll()` using the `request_token` returned from the run. Curious to get your thoughts.

Reply:
@bobzoller that's also the reason I put a `rpc.poll()` example below this one. Maybe I can be more explicit about that though since it clearly wasn't effective in this instance.